R/unused_import_linter.R

```diff
@@ -44,7 +44,7 @@
 unused_import_linter <- function(allow_ns_usage = FALSE, except_packages = c("bit64", "data.table", "tidyverse")) {
   # Get dataset names lazy-loaded by imported packages
   get_datasets <- function(pkg) {
```

Comment:
namespace_linter has `namespace_symbols()` already, taking lazydata into account.
We can probably refactor this and not use `utils::data()` at all.

Reply:
Since it's besides the point here, filed as #2080